<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow
{
	background:rgb(255, 85, 0)
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget
{
	background:rgb(255, 85, 0)
}</string>
   </property>
   <widget class="QPushButton" name="manageProductsButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>290</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: white;       /* Blue background */
    color: black;                    /* White text */
    border: 2px solid rgb(0, 0, 0);       /* Darker blue border */
    border-radius: 12px;             /* Rounded corners */
    font-size: 20px;                 /* Larger font */
    padding: 10px 20px;              /* Add some padding */
    min-width: 150px; 
}</string>
    </property>
    <property name="text">
     <string>Manage Products</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>390</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: white;       /* Blue background */
    color: black;                    /* White text */
    border: 2px solid rgb(0, 0, 0);       /* Darker blue border */
    border-radius: 12px;             /* Rounded corners */
    font-size: 20px;                 /* Larger font */
    padding: 10px 20px;              /* Add some padding */
    min-width: 150px; 
}</string>
    </property>
    <property name="text">
     <string>Finance Tab</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>390</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: white;       /* Blue background */
    color: black;                    /* White text */
    border: 2px solid rgb(0, 0, 0);       /* Darker blue border */
    border-radius: 12px;             /* Rounded corners */
    font-size: 20px;                 /* Larger font */
    padding: 10px 20px;              /* Add some padding */
    min-width: 150px; 
}</string>
    </property>
    <property name="text">
     <string>Reviews </string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>441</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>50</pointsize>
      <weight>7</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	color: white;
	font: 63 50pt &quot;Yu Gothic UI Semibold&quot;;
}</string>
    </property>
    <property name="text">
     <string>Store Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="manageOrdersButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>290</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: white;       /* Blue background */
    color: black;                    /* White text */
    border: 2px solid rgb(0, 0, 0);       /* Darker blue border */
    border-radius: 12px;             /* Rounded corners */
    font-size: 20px;                 /* Larger font */
    padding: 10px 20px;              /* Add some padding */
    min-width: 150px; 
}</string>
    </property>
    <property name="text">
     <string>Manage Orders</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>441</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>30</pointsize>
      <weight>7</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	color: white;
	font: 63 30pt &quot;Yu Gothic UI Semibold&quot;;
}</string>
    </property>
    <property name="text">
     <string>Pending Orders:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="numOrders">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>140</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>5</number>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
